1-select-tasks/15.txt

```diff
@@ -1,3 +1,21 @@
 Create a SQL query to return the total number of orders for every month in 1997 year:
 
-| January | February | March | April | May | June | July | August | September | November | December |
\ No newline at end of file
+| January | February | March | April | May | June | July | August | September | November | December |
+
+
+--task15
+SELECT 
+	SUM(CASE WHEN MONTH([o].[OrderDate]) = 1 THEN 1 ELSE 0 END) AS [january],
+    SUM(CASE WHEN MONTH([o].[OrderDate]) = 2 THEN 1 ELSE 0 END) AS [february],
+    SUM(CASE WHEN MONTH([o].[OrderDate]) = 3 THEN 1 ELSE 0 END) AS [march],
```

Comment:
it works, but do it with count function. Count function counts only fields that are not null)
https://stackoverflow.com/questions/17975229/using-sql-count-in-a-case-statement#answer-17975288